### PR TITLE
[Tizen/Platform] update a new application for the experiment

### PR DESCRIPTION
**Pull request by helloahn**
it constructs the multiple pipelines with vivante and tflite

see also: https://github.com/nnstreamer/nnstreamer/issues/2345

Signed-off-by: HyoungJoo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Name:		vivante-pipeline-experiment
+Summary:	sample-pipeline
+Version:	1.0.0
+Release:	0
+Group:		Applications/Multimedia
+Packager:	HyoungJoo Ahn <hello.ahn@samsung.com>
+License:	Priprietary
```
Comment (myungjoo) [inline Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec:7]: Update the License!

File: Tizen.platform/Vivante_pipeline_experiments/gen_vivante-pipeline-experiment_rpm.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+## @file gen_vivante_pipeline_example_rpm.sh
+## @author HyoungJoo Ahn <hello.ahn@samsung.com>
+## @date Feb 19 2020
+## @brief generate the Vivante Capi-pipeline example package
+##
+
+git init
+git add *
+git commit -m'Initial commit'
+gbs build -A armv7l --include-all
```
Comment (myungjoo) [inline Tizen.platform/Vivante_pipeline_experiments/gen_vivante-pipeline-experiment_rpm.sh:12]: In this case, you don't need --include-all

File: Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec (hunk 1 above)
Comment (helloahn) [reply Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec:7]: it's updated :)

File: Tizen.platform/Vivante_pipeline_experiments/gen_vivante-pipeline-experiment_rpm.sh (hunk 2 above)
Comment (helloahn) [reply Tizen.platform/Vivante_pipeline_experiments/gen_vivante-pipeline-experiment_rpm.sh]: you're right. it's removed!

File: Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec (hunk 1 above)
Comment (myungjoo) [reply Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec:7]: You need to release this as Apache 2.0. Please update it.

File: Tizen.platform/Vivante_pipeline_experiments/src/main.c [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+  }
+  if (enable_tflite_inception){
+    /* tflite inception v3 */
+    status = ml_pipeline_sink_register (handle, "tflite_inception_sink",
+      get_tflite_inception_result_from_tensor_sink, NULL,
+      &tflite_inception_sink_handle
+    );
+    if (status != ML_ERROR_NONE){
+      printf ("[%d] ERROR!!!: %s\n", __LINE__, getErrorName (status));
+      return 0;
```
Comment (dongju-chae) [inline Tizen.platform/Vivante_pipeline_experiments/src/main.c:237]: You may need to destroy resources on failures as well.

File: Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec (hunk 1 above)
Comment (helloahn) [reply Tizen.platform/Vivante_pipeline_experiments/packaging/vivante-pipeline-experiment.spec]: it's applied

File: Tizen.platform/Vivante_pipeline_experiments/src/main.c (hunk 3 above)
Comment (helloahn) [reply Tizen.platform/Vivante_pipeline_experiments/src/main.c]: I agree. the additional code has been updated.

**Pull request merged**